IntegrationTemplates/AnalyticsAndSIEM/AnalyticsAndSIEM.py

```diff
@@ -0,0 +1,443 @@
+        }
+        return self._http_request('GET', suffix, params=params)
+
+    def close_event_request(self, event_id: AnyStr) -> requests.Response:
+        """Gets events from given IDS
+
+        Args:
+            event_id: event to delete
+
+        Returns:
+            response json
+        """
+        # The service endpoint to request from
+        suffix: str = 'event'
+        # Dictionary of params for the request
+        params = {
+            'eventId': event_id
+        }
+        # Send a request using our http_request wrapper
+        return self._http_request('DELETE', suffix, params=params, resp_type='response')
+
+    def update_event_request(self, event_id: AnyStr, description: Optional[AnyStr] = None,
+                             assignee: Optional[List[str]] = None) -> Dict:
+        """Update given event
+
+        Args:
+            description: change description of event
+            assignee: User to assign event to
+            event_id: event ID
+
+        Returns:
+            response json
+        """
+        suffix: str = 'event'
+        params = {
+            'eventId': event_id,
+        }
+
+        if description:
+            params['description'] = description
+        if assignee:
+            params['assignee'] = assignee
+
+        return self._http_request('POST', suffix, params=params)
+
+    def create_event_request(self, description: str, assignee: List[str] = None) -> Dict:
+        """Update given event
+
+        Args:
+            description: change description of event
+            assignee: User to assign event to
+
+        Returns:
+            requests.Response
+        """
+        suffix = 'event'
+        params = {
+            'description': description,
+            'assignee': assignee
+        }
+
+        return self._http_request('POST', suffix, params=params)
+
+    def query_request(self, **kwargs):
+        suffix: str = 'query'
+        return self._http_request('GET', suffix, params=kwargs)
+
+
+''' HELPER FUNCTIONS '''
+
+''' COMMANDS '''
+
+
+def test_module(client: Client):
+    """
+    Performs basic get request to get item samples
+    """
+    if client.test_module():
+        demisto.results('ok')
+
+
+def fetch_incidents(client: Client):
+    """Uses to fetch credentials into Demisto
+    Documentation: https://github.com/demisto/content/tree/master/docs/fetching_credentials
+    """
+    # Get credentials from api
+    last_run = demisto.getLastRun()
+    if not last_run:  # if first time running
+        last_run, _ = parse_date_range(client.get_last_fetch())
+    raw_response: Dict = client.fetch_incidents(last_run)
+    events: List[Dict] = raw_response.get('incidents', [])
+    if events:
+        # Creates incident entry
+        incidents = [{
+            'name': event.get('title'),
+            'occurred': event.get('created'),
+            'rawJSON': json.dumps(event)
+        } for event in events]
+        demisto.setLastRun(datetime.now())
```

Comment:
use the time of the last event instead, it will cover us from some questions from users which will want to modify it a bit